index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Hana Sabih</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://code.iconify.design/iconify-icon/1.0.2/iconify-icon.min.js"></script>
+    <!-- <link href="https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp" rel="stylesheet"> -->
+  </head>
+  <body>
+    <header>
+      <h5>Welcome</h5>
+      <iconify-icon
+        icon="ic:round-menu"
+        width="24"
+        height="24"
+        class="menubar"
+      ></iconify-icon>
+    </header>
+    <section class="headline">
+      <h1>
+        Hey there. I'm Mariane<br />
+        <span> I'm a software developer</span>
+      </h1>
```

Comment:
- [ ] Please try to stick to Figma design as much as possible to make your project looks good, by following different properties like colors, fonts, etc.

Figma design:
![image](https://user-images.githubusercontent.com/90524466/205321950-5ae00cad-45d0-42d4-8131-a2ca36ab60ed.png)

Your design:
![image](https://user-images.githubusercontent.com/90524466/205322090-9c852394-21cb-484e-a02c-129a283d145f.png)